heron/tools/apiserver/src/java/com/twitter/heron/apiserver/resources/TopologyResource.java

```diff
@@ -502,7 +502,8 @@ private Config createConfig(Collection<Pair<String, Object>> keyValues,
   }
 
   private boolean isLocalMode() {
-    return "local".equalsIgnoreCase(getCluster()) || "standalone".equalsIgnoreCase(getCluster());
+    return "local".equalsIgnoreCase(getCluster()) || "standalone".equalsIgnoreCase(getCluster())
+        || "nomad".equalsIgnoreCase(getCluster());
```

Comment:
@jerrypeng - is this needed for 'nomad' direct deployment?

Reply:
no this is for API server

Reply:
ok got it - let us fix the CI and merge it.